rust_snuba/Cargo.toml

```diff
@@ -25,3 +25,6 @@ serde = { version = "1.0", features = ["derive"]}
 serde_json = { version = "1.0" }
 glob = "0.3.1"
 pyo3 = { version = "0.18.1", features = ["chrono", "extension-module"] }
+
+[dev-dependencies]
+chrono = "0.4.11"
```

Comment:
I needed this for the test since it's part of the Message interface in rust_arroyo. Though I wonder if we can get rid of it in arroyo too.

Reply:
I don't know at the moment, I don't think we need to solve this problem atm though. I am not sure if chrono is the best library for our needs